python/pyarrow/ipc.pxi

```diff
@@ -54,6 +54,14 @@ _WriteStats = namedtuple(
 
 class WriteStats(_WriteStats):
     """IPC write statistics
+
+    Parameters
+    ----------
+    num_messages : number of messages.
+    num_record_batches : number of record batches.
+    num_dictionary_batches : number of dictionary batches.
+    num_dictionary_deltas : delta of dictionaries.
+    num_replaced_dictionaries : number of replaced dictionaries.
```

Comment:
In practice those are never created by the user, so not sure how useful such a docstring is (except for passing the check ..)

Reply:
Although those can of course still be accessed by the user (so strictly speaking it might actually be more logical to list those as "Attributes" instead of "Parameters", but OK :))